### ISSUE TITLE
Drupal Local Development Guide might need updates?

### ISSUE THREAD

**Issue by brianyoungblood** [drud/ddev#3179]
**Describe the Problem**

When testing the [Local Development Guide](https://www.drupal.org/docs/official_docs/en/_local_development_guide.html) for Drupal, it mentions DDEV has Drush built-in. But that's actually not true. Prior to me creating a Drupal issue with the changes needed to explain that you must add Drush to your DDEV project with composer, I wanted to see if this is the correct way to explain it.

**To Reproduce**

After following the guide and starting the DDEV project. You'll then attempt to install Drupal, but find Drush isn't found.

```
ddev exec drush site-install --account-name=admin --account-pass=password
bash: drush: command not found
Failed to execute command drush site-install --account-name=admin --account-pass=password: exit status 127
```

**Proposed changes**

Next, start the DDEV project
```ddev start```

You now have a web server and database server configured and running.

Add Drush (a command-line utility for Drupal) to your DDEV project you just started.
```ddev composer require drush/drush```

Install Drupal
Next, you must install Drupal, which populates your Drupal application’s new database.

Using DDEV and Drush, execute the command below. Replace my-password with the password that you would like to use for the admin (user 1) account.

ddev exec drush site-install --account-name=admin --account-pass=my-password
Drupal is now installed.

**Comment by rfay** [comment]
Thanks for paying attention to that and working on maintaining it!

Drush 8 is provided in the ddev web container, but it's only linked as /usr/local/bin drush in D7 and D6, because the "official" drush recommendation is to site-install drush/drush. 

However, I've been intending to add a little guide to the docs to how to use drush8 on D8/9+, because it's easy.  Use `drush8` or add a symlink in a post-start hook or Dockerfile.

Overall, I probably think just adjusting the Local Devleopment Guide to match the [d9 quickstart](https://ddev.readthedocs.io/en/stable/users/cli-usage/#drupal-9-quickstart) is probably the thing, and it looks like you're on track there.

Thanks!

**Comment by brianyoungblood** [comment]
Thanks for the note @rfay. Do whatever you think should be done with the status of this issue here. I submitted the issue for the documentation update. https://www.drupal.org/project/official_docs/issues/3228626

Side note. I often poke around the documentation for doing new things I might share with others, and this happened to be one I came across. Although I do wonder if a link to DDEV docs would be a better way to go about helping so it's easier to keep updated. I'm sure other options for local development would then be represented and linking to those would be created, and in the end, the information is kept current. Thanks again, Randy.

**Comment by rfay** [comment]
I agree it would be better to just link to ddev docs... but not in charge of that page :) I'll follow the issue. 

> I often poke around the documentation for doing new things I might share with others, and this happened to be one I came across

I didn't understand this comment. Are you asking how you can share a tip/technique about ddev?

**Comment by brianyoungblood** [comment]
No. I look at Drupal.org documentation from time to time and was curious how others might find information on setting up local development. I'd think that a fair amount of people land on that Drupal.org page for this information.

**Comment by rfay** [comment]
Did you get this resolved on drupal.org @brianyoungblood ?

I wrote up how to use Drush 8 with Drupal8+ on https://stackoverflow.com/questions/69470610/how-can-i-use-drush-8-with-drupal-8-or-drupal-9
